layouts/test/single.html

```diff
@@ -1,65 +1,68 @@
 {{ define "main" }}
 <style>
     .container {
-        max-width: 780px;
+        max-width: 1200px;
+        width: 100%;
         margin: 0 auto;
-        min-height: 100vh;
+        padding: 0 16px; 
+        box-sizing: border-box;
     }
 
     .header {
-        border-bottom: var(--bs-border-width) solid var(--bs-border-color);
-        padding: 2rem 2rem 1.5rem;
+        border-bottom: 1px solid var(--bs-border-color);
+        padding: 24px 16px 20px;
     }
 
     .header h1 {
-        font-size: 1.75rem;
+        font-size: 1.5rem; 
         font-weight: 600;
         color: var(--bs-emphasis-color);
-        margin-bottom: 0.25rem;
+        margin-bottom: 8px; 
+        line-height: 1.334;
     }
 
     .header p {
         color: var(--bs-secondary-color);
-        font-size: 0.95rem;
+        font-size: 0.875rem;
+        line-height: 1.43; 
     }
 
     #go-back {
-        position: fixed; 
-        top: 160px;     
-        left: 25px;    
-        z-index: 1010; 
-        
-        width: 45px;      
-        height: 45px;      
-        background-color:white; 
-        border-radius: 50%;   
-        box-shadow: 0 4px 12px rgba(0, 0, 0, 0.15);
-        display: flex;      
+        position: fixed;
+        top: 160px;
+        left: 25px;
+        z-index: 1010;
+        width: 40px; 
+        height: 40px;
+        background-color: white;
+        border-radius: 50%;
+        box-shadow: 0 3px 5px -1px rgba(0, 0, 0, 0.2), 0 6px 10px 0 rgba(0, 0, 0, 0.14);
+        display: flex;
         justify-content: center;
-        align-items: center;    
+        align-items: center;
         color: var(--bs-primary);
-        font-size: 1.1rem;   
+        font-size: 1rem;
         cursor: pointer;
-        transition: all 0.2s ease-in-out; 
+        transition: all 0.2s ease-in-out;
         border: 1px solid var(--bs-border-color);
     }
 
     #go-back:hover {
-        transform: scale(1.1); 
-        box-shadow: 0 6px 16px rgba(0, 0, 0, 0.2);
+        transform: scale(1.1);
+        box-shadow: 0 5px 15px -3px rgba(0, 0, 0, 0.3);
         color: var(--bs-link-hover-color);
     }
 
     .form-container {
-        padding: 2rem;
+        padding: 24px 16px; 
     }
 
     .question {
         background: var(--bs-white);
-        border: var(--bs-border-width) solid var(--bs-border-color);
-        border-radius: var(--bs-border-radius);
-        padding: 1.5rem 1.75rem;
-        margin-bottom: 1.5rem;
+        border: 1px solid var(--bs-border-color);
+        border-radius: 4px;
+        padding: 16px; 
```

Comment:
Why has this been moved away from `rem`? Other than when sizing an SVG, we generally prefer use of `rem` to `px`. Is there a specific need to switch here?